src/test/scala/com/amazon/deequ/profiles/ColumnProfilerTest.scala

```diff
@@ -201,6 +201,151 @@ class ColumnProfilerTest extends WordSpec with Matchers with SparkContextSpec
       assert(histogram("NullValue").absolute == 1)
       assert(histogram("NullValue").ratio == 1.0 / nRows)
     }
+
+    "return histograms for IntegerType columns" in withSparkSession { session =>
+      val attribute = "attribute"
+      val nRows = 6
+      val data = com.amazon.deequ.dataFrameWithColumn(
```

Comment:
indent with 4 spaces, please

Reply:
Are you sure you want this? The project seems to be indented with 2 spaces everywhere, so I followed the style of 2 spaces.